.github/workflows/build_and_perform_instrumented_tests.yml

```
@@ -27,6 +27,9 @@ jobs:
           GOOGLE_SERVICES_JSON: ${{ secrets.GOOGLE_SERVICES_JSON }}
         run: echo $GOOGLE_SERVICES_JSON | base64 --decode > ./app/google-services.json
 
+      - name: Decode local properties
+        run: echo ${{ secrets.LOCAL_PROPERTIES }} | base64 --decode > ./local.properties
```

We can add `--ignore-garbage` to stay safe in case some future modification to the file causes the base64 string to have non-alphabet characters. For more info: https://linux.die.net/man/1/base64
```suggestion
        run: echo ${{ secrets.LOCAL_PROPERTIES }} | base64 --decode --ignore-garbage > ./local.properties
```

---

I cannot. I am getting `[base64: unrecognized option --ignore-garbage]` error